### PR TITLE
added assets folder

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
- [ ] Feature tasks for this challenge are completed
- [ ] Unit tests written and passing
- [ ] “Happy Path” - Expected outcome
- [ ] Expected failure
- [x]  Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
- [x] Summary, Description, Approach & Efficiency, Solution
- [x]  Link to code
- [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/leftJoin/README.md [hunk 1]
```diff
@@ -20,4 +20,4 @@ Combine the key and corresponding values (if they exist) into a new data structu
 
 ## Solution
 
-+ ![leftJoin](CC33.png)
\ No newline at end of file
++ ![leftJoin](assets/CC33.png)
```
Comment (adrianhuebner) [inline javascript/code-challenges/leftJoin/README.md:23]: Code looks good, just would like to see tests added!
